api/providers/serializers.py

```diff
@@ -138,6 +138,25 @@ class Meta:
         'name',
     ])
 
+    registrations = ReviewableCountsRelationshipField(
+        related_view='providers:registration-providers:registrations-list',
+        related_view_kwargs={'provider_id': '<_id>'},
+    )
+
+    links = LinksField({
+        'self': 'get_absolute_url',
+        'external_url': 'get_external_url',
+    })
+
+    def get_registrations_url(self, obj):
```

Comment:
Is this actually doing anything? I'm not seeing anything that would call this.

Reply:
no it is not, now that the 'registrations' (which were mislabeled as 'preprints') have been removed from the links. Thanks for the catch